### PR TITLE
Update the backup options after arc, lyra and elektra updated to helmcharts.

### PR DESCRIPTION
**Pull request by Dexus**
@databus23

### CODE REVIEW

File: openstack/elektra/values.yaml [hunk 1]
```diff
@@ -38,4 +38,19 @@ postgres:
     enabled: true
   name: elektra
 #   password:
+  backup:
+    enabled: false
+    repository: sapcc/backup-tools
+    image_version: v0.4.1
```
Comment (databus23) [inline openstack/elektra/values.yaml]: Should this be v0.4.6?

File: common/postgresql/templates/deployment.yaml [hunk 2]
```diff
@@ -53,8 +53,39 @@ spec:
...
+        env:
+          - name: MY_POD_NAME
+            value: postgres-backup-{{.Values.name}}
+          - name: MY_POD_NAMESPACE
+            value: {{.Release.Namespace}}
+          - name: OS_AUTH_URL
+            value: {{.Values.backup.os_auth_url}}
+          - name: OS_AUTH_VERSION
+            value: {{.Values.backup.os_auth_version | quote}}
+          - name: OS_IDENTITY_API_VERSION
```
Comment (databus23) [inline common/postgresql/templates/deployment.yaml]: What is the difference between `OS_AUTH_VERSION` and `OS_IDENTITY_API_VERSION`? Do we really need both?

File: common/postgresql/values.yaml [hunk 3]
```diff
@@ -46,3 +46,19 @@ resources:
   requests:
     memory: 256Mi
     cpu: 100m
+
+backup:
+  enabled: false
+  repository: sapcc/backup-tools
+  image_version: v0.4.6
+  interval_full: 3 days
+  interval_incr: 3 days
```
Comment (databus23) [inline common/postgresql/values.yaml]: Is it correct that these values are both set to 3 days?
Does this mean we only do an incremental backup every 3 days?

**Comment by Dexus** [comment]
- Backup updated to 3 hours
- `OS_AUTH_VERSION` and `OS_IDENTITY_API_VERSION` are similar as of review code of the swift-client. So removed `OS_IDENTITY_API_VERSION`.
- Update version of backup-tools.

**Pull request merged**